src/assets/styles/app.css

```diff
@@ -0,0 +1,265 @@
+:root {
+  --menu-bg-width: 50rem;
+  --menu-bg-width-phone: 320px;
```

Comment:
It would be nice to use the same units for the same things, but it works like this, too.